lego/apps/events/views.py

```diff
@@ -240,12 +240,23 @@ def admin_register(self, request, *args, **kwargs):
             raise APIEventNotFound()
         serializer = self.get_serializer(data=request.data)
         serializer.is_valid(raise_exception=True)
+
+        users = serializer.validated_data.pop('users')
```

Comment:
Maybe we should support passing in a single user as well? Then there is no "downtime" during deployment since the api will be backwards compatible.